ratpack/src/test/groovy/com/baeldung/RatpackGroovySpec.groovy

```diff
@@ -0,0 +1,47 @@
+package com.baeldung;
+
+import ratpack.groovy.Groovy
+import ratpack.groovy.test.GroovyRatpackMainApplicationUnderTest;
+import ratpack.test.http.TestHttpClient;
+import ratpack.test.ServerBackedApplicationUnderTest;
+import org.junit.Test;
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import ratpack.test.MainClassApplicationUnderTest
+
+class RatpackGroovySpec {
+
+    ServerBackedApplicationUnderTest aut = new MainClassApplicationUnderTest(RatpackGroovyApp.class)
+    @Delegate TestHttpClient client = TestHttpClient.testHttpClient(aut)
+
+    @Test
+    void "test if app is started"() {
+        when:
+        get("")
+
+        then:
+        response.statusCode == 200
+        assert response.body.text == "Hello World from Ratpck with Groovy!!"
```

Comment:
would we not use an assertions library? rather than just `assert`? I'm not sure what's conventional in Groovy? I'd rather hope there's something that does more than just `true` or `false` :)

Reply:
assert is by default available in Groovy spock framework and is more conventional. Also we are just comparing the text and status code so we don't need to have java like assert statements here.